src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { debounce } from 'lodash';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  delay: number;
+  onSelected: (person: Person) => void;
+};
+
+export const Autocomplete: React.FC<Props> = ({
+  people,
+  delay,
+  onSelected,
+}) => {
+  const [query, setQuery] = useState('');
+  const [showList, setShowList] = useState(true);
```

Comment:
It would be great to name boolean variables according to these rules:
![image](https://github.com/mate-academy/react_autocomplete/assets/96409563/cf6a9494-2894-4376-bf84-612226dcee58)
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig